app/models/post.rb

```diff
@@ -1,3 +1,15 @@
 class Post < ApplicationRecord
   belongs_to :user
+
+  def recent_comments(limit = 5)
+    comments.order(created_at: :desc).limit(limit)
+  end
+
+  def recent_posts
+    posts.order(created_at: :desc).limit(3)
+  end
+
+  def increment_posts_counter
+    update(posts_counter: posts_counter + 1)
+  end
```

Comment:
**OPTIONAL** - You did well so far, but this method will give an error as you have not specified what attribute you would like to update. Please fix this 🙏 
```ruby
 def increment_posts_counter
      user.posts_counter.update(user.posts_counter + 1)
 end
```

Reply:
done